sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/WindowQuerySuite.scala

```diff
@@ -59,56 +60,115 @@ class WindowQuerySuite extends QueryTest with SQLTestUtils with TestHiveSingleto
   }
 
   test("windowing.q -- 15. testExpressions") {
-    // Moved because:
-    // - Spark uses a different default stddev (sample instead of pop)
-    // - Tiny numerical differences in stddev results.
-    // - Different StdDev behavior when n=1 (NaN instead of 0)
-    checkAnswer(sql(s"""
-      |select  p_mfgr,p_name, p_size,
-      |rank() over(distribute by p_mfgr sort by p_name) as r,
-      |dense_rank() over(distribute by p_mfgr sort by p_name) as dr,
-      |cume_dist() over(distribute by p_mfgr sort by p_name) as cud,
-      |percent_rank() over(distribute by p_mfgr sort by p_name) as pr,
-      |ntile(3) over(distribute by p_mfgr sort by p_name) as nt,
-      |count(p_size) over(distribute by p_mfgr sort by p_name) as ca,
-      |avg(p_size) over(distribute by p_mfgr sort by p_name) as avg,
-      |stddev(p_size) over(distribute by p_mfgr sort by p_name) as st,
-      |first_value(p_size % 5) over(distribute by p_mfgr sort by p_name) as fv,
-      |last_value(p_size) over(distribute by p_mfgr sort by p_name) as lv,
-      |first_value(p_size) over w1  as fvW1
-      |from part
-      |window w1 as (distribute by p_mfgr sort by p_mfgr, p_name
-      |             rows between 2 preceding and 2 following)
+    withSQLConf(SQLConf.LEGACY_CENTRAL_MOMENT_AGG.key -> "true") {
```

Comment:
does this provider more test coverage than `DataFrameWindowFunctionsSuite` for this legacy config?

Reply:
I see your point. the legacy coverage code will only stay in DataFrameWindowFunctionsSuite, other UT will just be updated to expected answer.

Reply:
done.